src/Sarif/Schemata/sarif-schema.json

```diff
@@ -264,9 +264,9 @@
     "externalPropertyFile": {
```

Comment:
>externalPropertyFile [](start = 5, length = 20)

i officially do not care that this isn't renamed externalPropertyArtifact. i.e., any artifact which is guaranteed to be JSON is fine to be described as a file in my book. :) hope you agree.